codespell_lib/data/dictionary_en-GB_to_en-US.txt

```diff
@@ -32,7 +32,16 @@ criticises->criticizes
 crystallisation->crystallization
 customised->customized
 customising->customizing
+daemonisation->daemonization
+daemonisation->demonization
```

Comment:
Was the left-side supposed to be `demonisation` ?

Reply:
This conversion is already moved to #1653. I just messed up.

All corrections here wait for a custom aspell dictionary to pass the tests.

Reply:
I was more referring to the fact that you have two lines with `daemonisation` on the left side.

Reply:
Yes. You were right, it was supposed to be demonisation. but I moved the words, that are in aspell to #1653.
I deleted the duplicate entry now. :-)